indigo_app/views/tasks.py

```diff
@@ -107,6 +107,10 @@ def form_valid(self, form):
         response_object = super(TaskCreateView, self).form_valid(form)
         task = self.object
         task.workflows = form.cleaned_data.get('workflows')
+        if task.workflows.all():
```

Comment:
Each `.all()` will go to the database. So you can simply drop the `if` and rely on the fact that if there aren't workflows, the `for` loop does nothing.